client/src/app/site/pages/organization/pages/committees/pages/committee-detail/modules/committee-detail-edit/components/committee-detail-edit/committee-detail-edit.component.ts

```diff
@@ -89,11 +89,11 @@ export class CommitteeDetailEditComponent extends BaseComponent implements OnIni
      * @returns A function that will return a boolean
      */
     public getDisableOptionWhenFn(): (value: Selectable) => boolean {
-        return value => value.id === this.committeeId || !(this.isOrgaManager || this.committeeRepo.getViewModel(value.id).manager_ids?.includes(this.operator.user.id));
+        return value => (value.id === this.committeeId || this.isNotCommitteeAdminFor()(value));
     }
 
-    public getDisableOptionWhenNotComAdmin(): (value: Selectable) => boolean {
-        return value => !(this.isOrgaManager || this.committeeRepo.getViewModel(value.id).manager_ids?.includes(this.operator.user.id));
+    public isNotCommitteeAdminFor(): (value: Selectable) => boolean {
+        return value => !(this.isOrgaManager || this.operator.committeeCanManageNoOrgaCheck(value.id));
     }
```

Comment:
```suggestion
    public isNotCommitteeAdminFor: (value: Selectable) => boolean = value => !(
       this.isOrgaManager || this.operator.committeeCanManageNoOrgaCheck(value.id)
    );
```

That way you don't have to use double brackets for `getDisableOptionWhenFn`

Reply:
Same for `getDisableOptionWhenFn` tbh, there's no parameters that would change anything, so why use a function to return a function